components/console/helpers/questionhelper.rst

```diff
@@ -410,7 +410,7 @@ invalid answer and will only be able to proceed if their input is valid.
 
         $question = new Question('Please enter the name of the bundle', 'AcmeDemoBundle');
         $validation = Validation::createCallable(new Regex([
-            'pattern' => '/^[a-zA-Z]+Bundle$',
+            'pattern' => '/^[a-zA-Z]+Bundle$/',
```

Comment:
I suggest to use `'#^[a-zA-Z]+Bundle$#'` so that it doesn't look like an URI or file path.

Reply:
Yes I agree. Thanks @alexislefebvre  for the suggestion.